### DISCUSSION TITLE
Access Tracking Table data

### DISCUSSION THREAD

**Discussion by FarhadHussain21st** [ash-jc-allen/short-url#180] [Q&A] [answered]
Hi! 
I want to know that if there is any build in function to access the data of short_url_visits through ShortURL Elequent or we can use Where to the following table?

**Comment by ash-jc-allen** [comment, marked as answer]
Hey! The `AshAllenDesign\ShortURL\Models\ShortURL` class extends the `Illuminate\Database\Eloquent\Model` class, so you can interact with it like other models in your application. You can also use the `visits` method on the model to access the rows in the `short_url_visits` table that are associated with that Short URL (e.g. - `$shortUrl->visits`).

I hope this helps a bit? 😄

**Comment by FarhadHussain21st** [comment]
Hi Ash !

I am facing an issue when I share my link through whatsapp it does not track the platform and referal_url is null. Is there any option to track whatsapp as well? 

Thank you in advance.

**Reply by ash-jc-allen** [reply]
Hey @FarhadHussain21st! That's a great question, and I'm not entirely sure if it'd be possible to track if the messages were opened in an app such as WhatsApp.

I don't know if it'd fix your issue, but you could maybe create separate short URLs that are specifically sent in WhatsApp messages? 🙂
